### PR TITLE
Create incentiveproposal.md

### PR DESCRIPTION
**Pull request by Afauvre**
Draft and WIP. Will update mroe tomorrow.

### CODE REVIEW

**Comment by Afauvre** [comment]
Updated with feedback from @peterjgilbert and @pro-wh

File: src/operators/incentiveproposal.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Oasis Network Incentives: A proposal for the network at time of launch
+
+*NOTE: This is a draft proposal and is subject to change pending the outcome of an economic audit, feedback from the community, and the results of the Oasis Foundation’s staking competition.* 
+
+Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [slack](www.oasis-protocol.org/slack) or via [github](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
```
Comment (pro-wh) [inline src/operators/incentiveproposal.md]: https:// on slack link

File: src/operators/incentiveproposal.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+* **Transaction fee**:  The reward that a validator receives from processing a transaction that runs on the network (separate from staking rewards).
+
+* **Commission rate**: The fee that a validator chooses to charge to delegators.
+
+* **Unbonding period**: A period of time when a validator or delegator wants to stop staking tokens, but cannot move them. During this waiting period, tokens do not accrue staking rewards but cannot be withdrawn.
+
+*****
+For more information and for the latest updates, please visit our website. We also invite you to join our Slack community to share your feedback and help shape the future of our network. 
+If you’re new to Oasis, here are some resources you may find useful:
+* [Oasis Foundation Website](www.oasis-protocol.org)
```
Comment (pro-wh) [inline src/operators/incentiveproposal.md]: https:// on oasis foundation link

File: src/operators/incentiveproposal.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+* **Unbonding period**: A period of time when a validator or delegator wants to stop staking tokens, but cannot move them. During this waiting period, tokens do not accrue staking rewards but cannot be withdrawn.
+
+*****
+For more information and for the latest updates, please visit our website. We also invite you to join our Slack community to share your feedback and help shape the future of our network. 
+If you’re new to Oasis, here are some resources you may find useful:
+* [Oasis Foundation Website](www.oasis-protocol.org) 
+* [Testnet Documentation](https://docs.oasis.dev/operators/joining-the-testnet.html)
+* [Operator Documentation](https://docs.oasis.dev/operators/overview.html)
+* [GitHub](www.github.com/oasislabs)
```
Comment (pro-wh) [inline src/operators/incentiveproposal.md]: https:// on github link

File: src/operators/incentives-proposal.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
+# Oasis Network Incentives: A Proposal for the Network at Time of Launch
+
+*NOTE: This is a draft proposal and is subject to change pending the outcome of an economic audit, feedback from the community, and the results of the Oasis Foundation’s staking competition.* 
+
+Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [slack](www.oasis-protocol.org/slack) or via [github](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
```
Comment (peterjgilbert) [inline src/operators/incentives-proposal.md]:
```suggestion
Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [slack](https://www.oasis-protocol.org/slack) or via [github](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
```

File: src/operators/incentives-proposal.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+* **Transaction fee**:  The reward that a validator receives from processing a transaction that runs on the network (separate from staking rewards).
+
+* **Commission rate**: The fee that a validator chooses to charge to delegators.
+
+* **Unbonding period**: A period of time when a validator or delegator wants to stop staking tokens, but cannot move them. During this waiting period, tokens do not accrue staking rewards but cannot be withdrawn.
+
+*****
+For more information and for the latest updates, please visit our website. We also invite you to join our Slack community to share your feedback and help shape the future of our network. 
+If you’re new to Oasis, here are some resources you may find useful:
+* [Oasis Foundation Website](www.oasis-protocol.org)
```
Comment (peterjgilbert) [inline src/operators/incentives-proposal.md]:
```suggestion
* [Oasis Foundation Website](https://www.oasis-protocol.org) 
```

File: src/operators/incentives-proposal.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+* **Unbonding period**: A period of time when a validator or delegator wants to stop staking tokens, but cannot move them. During this waiting period, tokens do not accrue staking rewards but cannot be withdrawn.
+
+*****
+For more information and for the latest updates, please visit our website. We also invite you to join our Slack community to share your feedback and help shape the future of our network. 
+If you’re new to Oasis, here are some resources you may find useful:
+* [Oasis Foundation Website](www.oasis-protocol.org) 
+* [Testnet Documentation](https://docs.oasis.dev/operators/joining-the-testnet.html)
+* [Operator Documentation](https://docs.oasis.dev/operators/overview.html)
+* [GitHub](www.github.com/oasislabs)
```
Comment (peterjgilbert) [inline src/operators/incentives-proposal.md]:
```suggestion
* [GitHub](https://www.github.com/oasislabs)
```

File: src/operators/incentives-proposal.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+# Oasis Network Incentives: A Proposal for the Network at Time of Launch
+
+*NOTE: This is a draft proposal and is subject to change pending the outcome of an economic audit, feedback from the community, and the results of the Oasis Foundation’s staking competition.* 
+
+Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [slack](https://www.oasis-protocol.org/slack) or via [github](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
```
Comment (pro-wh) [inline src/operators/incentives-proposal.md]:
```suggestion
Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [Slack](https://www.oasis-protocol.org/slack) or via [GitHub](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
```

File: src/operators/incentives-proposal.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+Below is a proposal of the initial incentive model for the Oasis Network at the time of Mainnet launch. To submit your feedback and comments to the proposal please reach out to us on [Slack](https://www.oasis-protocol.org/slack) or via [GitHub](https://github.com/oasislabs/docs). When designing and running stress tests, audits, and simulations, the underlying goal has been to develop an incentive model that encourages both the development of a healthy and active developer ecosystem as well as a secure, decentralized network through an engaged and diverse node operator community.
+
+The core focus of this incentive documentation is to outline a proposal for parameters and rewards around staking and delegation. In future documentation we will provide more detail on the proposed roadmap and rewards that expand beyond this activity.
+
+## Summary
+
+* **Estimated Staking Rewards: 10-15% APR with higher rewards at network launch and tapering over time.**
+* **Slashing: Slash for double-signing only**
+* **Number of validators to participate in any given consensus committee (and receive staking rewards) at launch: up to 100 validators**
```
Comment (peterjgilbert) [inline src/operators/incentives-proposal.md]:
```suggestion
* **Number of validators to participate in the consensus committee (and receive staking rewards) at launch: up to 100 validators**
```

**Pull request merged**